Update diffraction scripts and group in same subfolder (#36)
scipp/ess-legacy: diffraction/README.md

# ess-legacy diffraction

This folder contains Python scripts and Jupyter notebooks to reduce diffraction 

> data with scipp and scippneutron.

Reviewer: ```suggestion
data with `scipp` and `scippneutron`.
```

> The main script is `powder_reduction.py` with used event data from WISH at ISIS STFC.

Reviewer: ```suggestion
The main script is `powder_reduction.py` which uses event data from WISH at ISIS STFC.
```

> The required files i.e. sample, vanadium, empty instrument and calibration files are not provided.

Reviewer: Could these files be shared publicly? If so, it might be nice to make them available.
We are also having to host more and more files for our neutron docs etc...
Maybe we should have a DRAM-wide discussion to try and find a solution for this?
Maybe like Mantid's FTP server for data files?

Note that it does not reproduce the official data reduction for this instrument. But it 

> simply illustrates how to use scipp and scippneutron to perform some of the steps required

Reviewer: ```suggestion
simply illustrates how to use `scipp` and `scippneutron` to perform some of the steps required
```

to reduce data from event NeXus files collected at a spallation source.

The Jupyter notebooks are used to demonstrate the implementation of intermediate steps of
the reduction.

> The legacy folder contains older scripts, which are likeliy not to run with the current 

Reviewer: ```suggestion
The legacy folder contains older scripts, which will most likely not run with the current 
```
